onair/data_handling/csv_parser.py

```diff
@@ -27,14 +27,22 @@ def process_data_file(self, data_file):
 ##### INITIAL PROCESSING ####
     def parse_csv_data(self, data_file):
         #Read in the data set
-        dataset = pd.read_csv(data_file, delimiter=',', header=0, dtype=str)
-        dataset = dataset.loc[:, ~dataset.columns.str.contains('^Unnamed')]
 
-        #Initialize the entire data dictionary
         all_data = []
-        for index, row in dataset.iterrows():
-            rowVals = floatify_input(list(row))
-            all_data.append(floatify_input(list(row)))
+
+        with open(data_file, 'r', newline='') as csv_file:
+            dataset = csv.reader(csv_file, delimiter=',')
+
+            #Initialize the entire data dictionary
+            index = 0
+            for row in dataset:
+                if index == 0:
+                    # Skip first row (headers)
+                    pass
+                else:
+                    rowVals = floatify_input(list(row))
+                    all_data.append(rowVals)
+                index = index + 1
```

Comment:
Not a show stopper, but if the only reason we are tracking index is to skip the first row, we can use the `next()` func on the iterator:
```
next(dataset) # skips header, StopIteration error if empty
```
or
```
next(dataset, None) # skips header, returns None if empty
```
this would remove the need for index check and count
```suggestion
            next(dataset) # skips header, StopIteration error if empty
            for row in dataset:
                rowVals = floatify_input(list(row))
                all_data.append(rowVals)
```

Reply:
Thanks for the tip! I'll definitely use it next time around